pkg/orchestrator/provider/common/utils.go

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/sirupsen/logrus"
+
+	familiestypes "github.com/openclarity/vmclarity/pkg/shared/families/types"
+)
+
+// scanSizesGB represents the memory sizes on the machines that the tests were taken on.
```

Comment:
I think that this section (up to type LogarithmicFormula struct {) should be in the aws package